install/install.sh

```diff
@@ -151,12 +151,15 @@ sed -i '$ a noipv4ll' /etc/dhcpcd.conf
 
 echo "Downloading bootstrap"
 COMPANION_BOOTSTRAP="bluerobotics/companion-bootstrap:master" # We don't have others tags for now
+BLUEOS_CORE="bluerobotics/companion-core:$VERSION" # We don't have a stable tag yet
+
 docker pull $COMPANION_BOOTSTRAP
-# Start bootstrap for the first time to fetch the other images and allow docker to restart it after reboot
-docker run \
+docker pull $BLUEOS_CORE
+# Create blueos-bootstrap container
+docker create \
```

Comment:
What is the problem with docker run here to do the register done by https://github.com/bluerobotics/BlueOS-docker/pull/812/commits/b0449894a1330fff32c0af6bf0f736ea96361430 ?

Reply:
this is just cleaner, as we don't need to kill bootstrap or have it kill itself in the first run.
this fixes https://github.com/bluerobotics/BlueOS-docker/issues/784, btw